README.md

```diff
@@ -1,3 +1,38 @@
-# OpenFE
+[![build](https://github.com/OpenFreeEnergy/openfe/actions/workflows/ci.yaml/badge.svg)](https://github.com/OpenFreeEnergy/openfe/actions/workflows/ci.yaml)
+[![coverage](https://codecov.io/gh/OpenFreeEnergy/openfe/branch/main/graph/badge.svg)](https://codecov.io/gh/OpenFreeEnergy/openfe)
 
-The best software ever written. Obviously!
+# The Open Free Energy library
+
+Alchemical free energy calculations for the masses.
+
+See our [website](https://openfree.energy/) for more information.
+
+# License
+
+This library is made available under the [MIT](https://opensource.org/licenses/MIT) open source license.
+
+# Important note
+
+This is pre-alpha work, it should not be considered ready for production and API changes are expected at any time without prior warning.
+
+# Install
+
+Dependencies can be installed via conda through:
+
+```bash
+conda env create -f environment.yml
+```
+
+The openfe library can be installed via:
+
+```
+pip install --no-deps .
```

Comment:
```suggestion
python -m pip install --no-deps .
```

Worked for me like this. I think the only difference between `python -m pip` and `pip` is that the first uses your Python environment to find `pip`, while the second uses `$PATH`. The Python environment is considered more trustworthy.

```
(tmp) Yvette:~/omsf/src/openfe dwhs$ pip list                               (skeleton)✱
Package    Version
---------- -------
pip        21.3.1
setuptools 60.5.0
wheel      0.37.1
(tmp) Yvette:~/omsf/src/openfe dwhs$ python -m pip install --no-deps .      (skeleton)✱
Processing /Users/dwhs/Dropbox/omsf/src/openfe
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: openfe
  Building wheel for openfe (setup.py) ... done
  Created wheel for openfe: filename=openfe-0.0.1.dev0-py2.py3-none-any.whl size=2890 sha256=0adcfad60964f610bbfde7c9bf17a39b78f39b7b8f2735aa0b232823b35c215f
  Stored in directory: /private/var/folders/vj/28c107496sq5z4y10rjz5gdm0000gn/T/pip-ephem-wheel-cache-xznd4o5z/wheels/27/b9/3e/f701d7ab1ab4aa2488af79a7ae64d2058ab0d1bc3508bc195d
Successfully built openfe
Installing collected packages: openfe
Successfully installed openfe-0.0.1.dev0
```

Reply:
Wait, may have spoken too soon -- it says it installed, but it doesn't import correctly. Looking into it.

Reply:
It's a problem with a setup.cfg. I'll fix it in a PR that follows up on this one.